bullet/src/main/java/org/bytedeco/bullet/presets/LinearMath.java

```diff
@@ -148,6 +148,8 @@ public void map(InfoMap infoMap) {
 
             .put(new Info("btAlignedObjectArray.h").linePatterns("\tclass less", "\t};").skip())
 
+            .put(new Info("btDefaultMotionState").virtualize())
+            .put(new Info("btMotionState").virtualize())
             .put(new Info("btIDebugDraw").virtualize())
```

Comment:
Let's put all that on the same line like this:
```
            .put(new Info("btIDebugDraw", "btMotionState", "btDefaultMotionState").virtualize())
```
And let's also add a line to CHANGELOG.md like this:
```
 * Virtualize `btMotionState` and `btDefaultMotionState` from Bullet Physics SDK to allow customization ([pull #1297](https://github.com/bytedeco/javacpp-presets/pull/1297))
```

Reply:
Done!